Scutea Emilia/Lab 9/main.c

```diff
@@ -0,0 +1,108 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+void extractSubstring(const char *string1[])
```

Comment:
I don't understand, why do you use both a pointer and `[]`. Are you trying to send a matrix in the function?